### PR TITLE
[IOTDB-384] Fix main feature text style in homepage

### PR DESCRIPTION
**Pull request by samperson1997**
Change main feature text style in homepage by removing `text-align: justify`.

Before:
截屏2019-12-2320 45 41

After:
截屏2019-12-2320 45 33

### CODE REVIEW

**Comment by samperson1997** [comment]
> Thanks for the PR @samperson1997 !
> 
> Hereby three suggestions:
> 
> * It's better to contain JIRA. number in the PR title. such as: [IOTDB-384] ix main feature text style in homepage.
> * The commit message should contain the JIRA. number such as : [IOTDB-384] ix main feature text style in homepage.
>   Then the JIRA and the PR can make the relation automatically
> * It's better to fix the problem of "Latest News "  in this PR.
> 
> What do you think ? :)
> 
> Best,
> Jincheng

Hi @sunjincheng121 , sorry for somehow ignoring the "latest news" part... and many thanks for the suggestions! I will notice the PR and commit format in the future : )

File: src/components/SideBar.vue [hunk 1]
```diff
@@ -68,7 +83,7 @@
   }
 
   .btn-white:hover {
-    background-color: rgb(238,163, 69);
+    background-color: rgb(238, 163, 69);
```
Comment (sunjincheng121) [inline src/components/SideBar.vue]: Good catch~

File: src/components/SideBar.vue [hunk 2]
```diff
@@ -1,17 +1,27 @@
 <template>
   <div class="col-sm-3  blog-sidebar">
     <div class="github-button-group">
-      <github-button class="github-button" href="https://github.com/apache/incubator-iotdb/fork" data-icon="octicon-repo-forked" data-size="large" data-show-count="true" aria-label="Fork apache/incubator-iotdb on GitHub">Fork</github-button>
-      <github-button class="github-button" href="https://github.com/apache/incubator-iotdb" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star apache/incubator-iotdb on GitHub">Star</github-button>
+      <github-button class="github-button" href="https://github.com/apache/incubator-iotdb/fork"
```
Comment (sunjincheng121) [inline src/components/SideBar.vue]: Is this a necessary change? Is only a code format improvement? , Just suggest that we can do this change in another PR. :) , but  I am fine with this change if you really want do that.

**Comment by sunjincheng121** [comment]
Thanks for the quick update @samperson1997 👍 
I only left one question, otherwise, the PR looks pretty good to me! :)

+1 to be merged from my side :)

Best,
Jincheng

File: src/components/SideBar.vue (hunk 2 above)
Comment (samperson1997) [reply src/components/SideBar.vue]: Ah... it's just an auto format from my IDE🙈...

**Comment by jixuan1989** [comment]
Thanks for reviewing @sunjincheng121

File: src/components/SideBar.vue (hunk 2 above)
Comment (sunjincheng121) [reply src/components/SideBar.vue]: I see, It would be better if we have common code style for all of the IoTDB contributors. :)
